src/ol/worker/version.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @module ol/worker/version
+ * A worker that responds to messages by posting a message with the version identifer.
+ */
+import {VERSION} from '../util';
+
+onmessage = event => {
+  console.log('version worker received message:', event.data); // eslint-disable-line
+  // @ts-ignore
+  postMessage(`version: ${VERSION}`);
+};
+
+export let create;
```

Comment:
This is here to satisfy our eslint/import rule.  After the worker is "built" it exports a named `create` function.